django/contrib/sitemaps/__init__.py

```diff
@@ -122,6 +124,14 @@ def location(self, item):
 
     def get_protocol(self, protocol=None):
         # Determine protocol
+        if not any((self.protocol, protocol)):
```

Comment:
`any()` is nice but it's over-engineering for me, we don't need to create a tuple and use `any()` to check two values.
```suggestion
        if self.protocol is None and protocol is None:
```

Reply:
Thanks for being more explicit about this than I was! (https://github.com/django/django/pull/14396#discussion_r632488901)